sunnypilot/mads/helpers.py

```diff
@@ -55,7 +54,7 @@ def set_car_specific_params(self, CP, CP_SP):
       hyundai_cruise_main_toggleable = True
       if hyundai_cruise_main_toggleable:
         CP_SP.flags |= HyundaiFlagsSP.LONGITUDINAL_MAIN_CRUISE_TOGGLEABLE.value
-        CP.safetyConfigs[-1].safetyParam |= HyundaiSafetyFlags.FLAG_HYUNDAI_LONG_MAIN_CRUISE_TOGGLEABLE.value
+        CP_SP.safetyParam |= HyundaiSafetyFlagsSP.LONG_MAIN_CRUISE_TOGGLEABLE
```

Comment:
```suggestion
        CP_SP.safetyParam |= HyundaiSafetyFlagsSP.LONG_MAIN_CRUISE_TOGGLEABLE.value
```
I am pretty sure we need to use `.value`, dont we?

Reply:
For `HyundaiSafetyFlagsSP`, since we defined them directly as integer constants (not as an Enum), we don't need the `.value`. The constant `LONG_MAIN_CRUISE_TOGGLEABLE` is already the integer value that can be used directly in bitwise.